exercises/Euclidean/python/main.py

```diff
@@ -0,0 +1,65 @@
+def get_nums():
+    while(True):
+        try:
+            num1 = int(input("Please enter a positive integer: "))
+            break
+        except TypeError:
+            pass
+    while(True):
+        try:
+            num2 = int(input("Please enter a positive integer: "))
+            break
+        except TypeError:
+            pass
+
+    return [num1,num2]
+
+def euclid_loop(x,y,prev):
```

Comment:
comment doc for these functions describing their expected functionality would be nice